api/src/organizational-entities/domain/usecases/archive-certification-centers-in-batch.usecase.js

```diff
@@ -0,0 +1,35 @@
+import { ArchiveCertificationCentersInBatchError } from '../errors.js';
+
+/**
+ * @param {Object} params
+ * @param {Array} params.certificationCenterIds
+ * @param {Number} params.userId
+ * @param {Object} params.certificationCenterForAdminRepository
+ * @param {Object} params.certificationCenterApiRepository
+ */
+export const archiveCertificationCentersInBatch = async function ({
```

Comment:
C'est un usecase de modification, aussi ne devrait-il pas être transactionnel avec `withTransaction` ?

Reply:
Bien reçu, compris. Il serait bien que cette information soit documentée d'une manière ou d'une autre pour que des maniacs de la généralisation ne viennent pas plus tard ajouter un `withTransaction` … 😉

Reply:
Est-ce que la bonne approche ne serait pas plutôt de mettre un `withTransaction` au usecase et de faire des commits de transaction dans le `for` ? Ainsi on aurait mieux que de la doc, on aurait du code qui parlerait par lui-même.

Reply:
Bien reçu. On peut alors juste mettre un petit commentaire dans ce usecase et nous organiserons un atelier quand nous aurons toute l'équipe pour définir notre stratégie sur ce sujet.